Source/Bifrost.Events.Azure.Tables/EventStore.cs

```diff
@@ -71,7 +71,7 @@ public void Commit(IEnumerable<EventAndEnvelope> eventsAndEnvelopes)
                 batch.Add(TableOperation.Insert(@event));
             });
 
-            _table.ExecuteBatchAsync(batch);
+            _table.ExecuteBatchAsync(batch).Wait();
```

Comment:
If the eventstore fails to commit the events, and no exception is handled, will not that mean that the events are published, and read models updated but the resulting read model states can never be rehydrated from the event store, since the events no longer exists?